release_bot/releasebot.py

```diff
@@ -81,9 +81,24 @@ def find_open_release_issues(self):
             else:
                 for edge in reversed(edges):
                     cursor = edge['cursor']
-                    match = re.match(r'(.+) release', edge['node']['title'].lower())
+                    last_version = Version(self.github.latest_release())
```

Comment:
`last_version` -> `latest_version`

('last' means there will be no more releases)